### ISSUE TITLE
Click() on login button not working for IE 11

### ISSUE THREAD

**Issue by woahitsjc** [angular/protractor#4535]
Trying to login by using 3 different types of clicks to click on the login button, all of them are not working but if I use `sendKeys(protractor.Key.ENTER);` that will work. Click works for Chrome.

- Node Version: `8.4`
- Protractor Version: `5.1.2`
- Browser(s): `IE 11`
- Operating System and Version `Windows 10`
- Your protractor configuration file

```
// An example configuration file.
exports.config = {

  seleniumAddress: 'http://localhost:4444/wd/hub',

  // Capabilities to be passed to the webdriver instance.
  capabilities: {
    'browserName': 'internet explorer',
    'nativeEvents': false,
    'unexpectedAlertBehaviour': 'accept',
    'ignoreProtectedModeSettings': true,
    'enablePersistentHover': true,
    'disable-popup-blocking': true
  },

  // Framework to use. Jasmine is recommended.
  framework: 'jasmine',

  // Spec patterns are relative to the current working directory when
  // protractor is called.
  specs: ['loginProt.js'],
};
```

- A relevant example test

```
describe('Selenium Test Case', function() {
  var EC = protractor.ExpectedConditions;

  beforeEach(function () {
    //Experian is not an angular app
    browser.waitForAngularEnabled(false);
    browser.get("https://uat.backgroundchecking.com/app");
  });

  it('should execute test case without errors', function() {
    var text, value, bool, source, url, title;
    var TestVars = {};

    //Get all the inputs made by a user
    element(by.css("#Email")).sendKeys("username");
    element(by.css("#Password")).sendKeys("password");
    var button = element(by.css(".ui-button"));
    button.click();
    // browser.actions().doubleClick(button).perform();
    // browser.executeScript("var tmp = arguments[0]; tmp.click()", button);
    text = element(by.css("#content > h2")).getText();
    expect(text).toContain("Administrator Home Page");
  });
});
```

- Output from running the test

```
Failures:
1) Selenium Test Case should execute test case without errors
  Message:
    Expected 'Login' to contain 'Administrator Home Page'.
  Stack:
    Error: Failed expectation
        at UserContext.<anonymous> (C:\Users\JustinChew\Documents\Selenium2Testing\spec\loginProt.js:22:18)
        at C:\ProgramData\nvm\v8.4.0\node_modules\protractor\node_modules\jasminewd2\index.js:112:25
        at new ManagedPromise (C:\ProgramData\nvm\v8.4.0\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1067:7)
        at ControlFlow.promise (C:\ProgramData\nvm\v8.4.0\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2396:12)
        at schedulerExecute (C:\ProgramData\nvm\v8.4.0\node_modules\protractor\node_modules\jasminewd2\index.js:95:18)
        at TaskQueue.execute_ (C:\ProgramData\nvm\v8.4.0\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2970:14)
        at TaskQueue.executeNext_ (C:\ProgramData\nvm\v8.4.0\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2953:27)
        at asyncRun (C:\ProgramData\nvm\v8.4.0\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2860:25)
        at C:\ProgramData\nvm\v8.4.0\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:676:7
```

- Steps to reproduce the bug
`webdriver-manager update --ie32 --versions.standalone=3.4.0 --versions.ie=3.5.1`
`webdriver-manager start --versions.standalone=3.4.0`
`protractor conf.js`

**Comment by wswebcreation** [comment]
Hi @woahitsjc 

I'm not able to see the source code (on my mobile now) but it the css-selector also the element that holds the click-event? I saw this also in previous tests and chrome excepts clicks on elements that don't hold the real click event (it bubble's it down) and in other browsers you need to select the element that really holds the click.

Can you verify that?

**Comment by woahitsjc** [comment]
Hi @wswebcreation 

I inspected both IE and Chrome elements and there doesn't seem to be another element that holds the click-event. 

<input type="submit" value="Login" class="ui-button ui-widget ui-state-default ui-corner-all" role="button" aria-disabled="false">

This is the only element that the button points to.

**Comment by woahitsjc** [comment]
Solved the issue, had a look at [#2244](https://github.com/angular/protractor/issues/2244) global windows display settings was set at 125%. Setting it to 100% ensure it clicked on the button.

**Comment by wswebcreation** [comment]
Cool

Tnx for the feedback
